src/contconic.jl

```diff
@@ -1,6 +1,23 @@
 using MathOptInterfaceUtilities # Defines getindex for VectorAffineFunction
 const MOIU = MathOptInterfaceUtilities
 
+_vovf(vecofvars) = vecofvars ? MOI.VectorOfVariables : MOI.VectorAffineFunction{Float64}
+"""
+    _addvovconstraint!(instance, v::Vector{MOI.VariableIndex}, set::MOI.AbstractVectorSet, vecofvars::Bool)
```

Comment:
The underscore in the name here breaks code highlighting for the rest of the file.